terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,5 @@
+environment = "cidev"
+aws_profile = "development-eu-west-2"
+
+# service configs
+use_set_environment_files = true
```

Comment:
need to add schedule to cidev profile (doing this now for consumer services)

# Scheduled scaling of tasks
service_autoscale_enabled  = true
service_scaledown_schedule = "55 19 * * ? *"
service_scaleup_schedule   = "5 6 * * ? *"